packages/dd-trace/src/proxy.js

```diff
@@ -25,7 +25,7 @@ class Tracer extends NoopProxy {
     try {
       const config = new Config(options) // TODO: support dynamic config
 
-      if (!config.isCiVisibility) {
+      if ((!config.remoteConfig || config.remoteConfig.enabled !== false) && !config.isCiVisibility) {
```

Comment:
the remoteConfig is enabled by default (when `config= {}` or `config.remoteConfig={}`) unless explicitly turned off.